packages/tokens/src/Tokens.story.tsx

```diff
@@ -61,17 +60,17 @@ const spacingBlockVariants = Object.keys(spacing).reduce(
 function SpacingBlock({ space }: { space: keyof typeof spacing }) {
   return (
     <div className={gutter}>
-      <InlineCode>
+      <code>
```

Comment:
Why the switch on all of these elements?

Reply:
was trying to avoid a circular dependency

Reply:
Where Tokens <> Typography both reference each other in their package.json files